docs/src/app/components/pages/components/refresh-indicator.jsx

```diff
@@ -0,0 +1,82 @@
+let React = require('react');
+let { RefreshIndicator } = require('material-ui');
+let ComponentDoc = require('../../component-doc');
+
+
+let RefreshIndicatorPage = React.createClass({
+
+  render() {
+
+    let code =
+      '// Ready status \n' +
+      '<RefreshIndicator percentage={30} size={40} left={10} top={5} status="ready" />\n' +
+      '<RefreshIndicator percentage={60} size={40} left={10} top={5} status="ready" />\n' +
+      '<RefreshIndicator percentage={80} size={40} left={10} top={5} status="ready" />\n' +
+      '<RefreshIndicator percentage={100} size={40} left={10} top={5} status="ready" />\n' +
+      '// Loading status \n' +
+      '<RefreshIndicator size={40} left={80} top={5} status="loading" />\n';
+
+    let componentInfo = [
+      {
+        name: 'Props',
+        infoArray: [
+          {
+            name: 'size',
+            type: 'number',
+            header: 'default: 40',
+            desc: 'Size in pixel.'
+          },
+          {
+            name: 'percentage',
+            type: 'number',
+            header: 'default: 0',
+            desc: 'The confirmation progress to fetch data. Max value is 100'
```

Comment:
Thank you very much for your patience.

In fact, It can be set `status` to be `loading` at any percentage in the container component. The defination is my own opinion, and the arrow will be highlighted when percentage equals 100.

Android's [SwipeRefreshLayout](https://developer.android.com/reference/android/support/v4/widget/SwipeRefreshLayout.html) component has `setDistanceToTriggerSync` and `setRefreshing` methods. They almost have the same meaning with my implementation.

The difference between my implementation and specs is the highlight behavior. I add it because I think it can make much sense to the user.